chall03/mlindhol.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func error() {
+	fmt.Printf("whoops, error!\n")
+	os.Exit(1)
+}
+
+func main() {
+
+	resp, err := http.Get("https://chall03.hive.fi/")
+	if err != nil {
+		error()
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		error()
+	}
+
+	str := strings.Split(string(body), ",")
+
+	reg, err := regexp.Compile("[^0-9]+")
```

Comment:
You should avoid using regex: https://softwareengineering.stackexchange.com/a/113243